### PR TITLE
Add export feature implementation details in DG

### LINKED ISSUE #76: Add design consideration for export feature in DG
Add the design considerations for export feature into DG

### PR DESCRIPTION
**Pull request by yanchenglee98**
Add export feature implementation details in DG under section 3 Implementation

### CODE REVIEW

**Review by schoolex — COMMENTED** [review]
Should we delete and replace 3.1 with the exporting information?

**Comment by yanchenglee98** [comment]
> Should we delete and replace 3.1 with the exporting information?

Redo/undo section has been removed from DG

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -191,90 +191,17 @@ Classes used by multiple components are in the `seedu.AddressBook.commons` packa
...
-
-_{more aspects and alternatives to be added}_
-
-### 3.2 \[Proposed\] Data archiving
-
-_{Explain here how the data archiving feature will be implemented}_
-
+##### Aspect: Use of .txt to export the information
+* We decided to use .txt for exporting as it is more user-friendly. Most users regardless of technical skills should know how to open a .txt file.
+* .txt files are can be easily opened on most operating systems e.g. MacOS, Windows.
```
Comment (schoolex) [inline docs/DeveloperGuide.md]: Seems like there is a typo here

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -191,90 +191,17 @@ Classes used by multiple components are in the `seedu.AddressBook.commons` packa
...
-  * Cons: We must ensure that the implementation of each individual command are correct.
-
-_{more aspects and alternatives to be added}_
-
-### 3.2 \[Proposed\] Data archiving
-
-_{Explain here how the data archiving feature will be implemented}_
-
+##### Aspect: Use of .txt to export the information
+* We decided to use .txt for exporting as it is more user-friendly. Most users regardless of technical skills should know how to open a .txt file.
```
Comment (schoolex) [inline docs/DeveloperGuide.md]: should it be .txt files instead?

**Review by schoolex — COMMENTED** [review]
Almost there, some nits to fix

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (yanchenglee98) [reply docs/DeveloperGuide.md]: Ok done

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yanchenglee98) [reply docs/DeveloperGuide.md]: Good catch. Thanks

**Review by schoolex — APPROVED** [review]

**Pull request merged**